data/lib/api/match/match_model.dart

```diff
@@ -209,3 +209,27 @@ enum TossDecision {
 
   const TossDecision(this.value);
 }
+
+@freezed
+class MatchResult with _$MatchResult {
+  const factory MatchResult({
+    @Default(0) int win,
+    @Default(0) int tie,
+    @Default(0) int lost,
+  }) = _MatchResult;
+}
+
+@freezed
+class MatchStats with _$MatchStats {
```

Comment:
this is team stat right? it's about team's played matches and its results, can we name it related to **team** and not **match**.